Library/Formula/vc4asm.rb

```diff
@@ -0,0 +1,27 @@
+class Vc4asm < Formula
+  homepage "http://maazl.de/project/vc4asm/doc/index.html"
+  desc "Macro assembler for Broadcom VideoCore IV aka Raspberry Pi GPU"
```

Comment:
It should be above the homepage. Sorry for not mentioning this in my previous comment.